### PR TITLE
[ops] Creates prometheus endpoint and add cors

### PR DESCRIPTION
**Pull request by chrisvrose**
- Added build info and git properties. This way, we populate the info endpoint with our current build.
- Add cors. For local, a configuration is exposed. Otherwise, a static list is used.
- Add a prometheus endpoint. It requires credentials right now.
- Add `@Timed` annotations to get metrics on every endpoint

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -1,65 +1,72 @@
...
-	testRuntimeOnly 'com.h2database:h2'
+    compileOnly 'org.projectlombok:lombok'
+    implementation 'org.springframework.boot:spring-boot-starter-actuator'
+    implementation 'org.springframework.boot:spring-boot-starter-data-jpa'
+    implementation 'org.springframework.boot:spring-boot-starter-security'
+    implementation 'org.springframework.boot:spring-boot-starter-web'
+    implementation 'org.springframework.boot:spring-boot-starter-validation'
+    implementation 'com.auth0:java-jwt:4.4.0'
+    implementation 'org.modelmapper:modelmapper:3.2.0'
+    runtimeOnly 'io.micrometer:micrometer-registry-prometheus'
```
Comment (chrisvrose) [inline build.gradle:45]: new dependency

File: build.gradle [hunk 2]
```diff
@@ -1,65 +1,72 @@
...
-	annotationProcessor 'org.projectlombok:lombok'
-	testImplementation 'org.springframework.boot:spring-boot-starter-test'
-	testImplementation 'org.springframework.security:spring-security-test'
-	testRuntimeOnly 'com.h2database:h2'
+    compileOnly 'org.projectlombok:lombok'
+    implementation 'org.springframework.boot:spring-boot-starter-actuator'
+    implementation 'org.springframework.boot:spring-boot-starter-data-jpa'
+    implementation 'org.springframework.boot:spring-boot-starter-security'
+    implementation 'org.springframework.boot:spring-boot-starter-web'
+    implementation 'org.springframework.boot:spring-boot-starter-validation'
```
Comment (chrisvrose) [inline build.gradle:42]: new dependency, planned for future use.

**Review by SundarakrishnanN — APPROVED** [review]
LGTM

File: build.gradle (hunk 2 above)
Comment (chrisvrose) [reply build.gradle]: Deleted

**Pull request merged**